src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  for (const action of actions) {
+    const { type, extraData, keysToRemove } = action;
```

Comment:
you can do it directly in loop like 

const { type, extraData, keysToRemove } of actions